libraries/AP_RangeFinder/AP_RangeFinder_LightWareI2C.cpp

```diff
@@ -467,21 +467,45 @@ void AP_RangeFinder_LightWareI2C::update(void)
 
 void AP_RangeFinder_LightWareI2C::legacy_timer(void)
 {
+    static const uint32_t TIMER_FREQ = 20;
```

Comment:
Is there a reason to not reduce the period of the timer itself and instead use this static counter?